cdk-testsuite/config/config.yaml

```diff
@@ -0,0 +1,26 @@
+path_linux : '/home/amit/openshift/rc5/cdk/components/rhel/misc/shared_folder/rhel-ose'
+path_win : ''
+path_mac : ''
+openshift_lib_MODULE : '/home/amit/workspace/cdk-testsuite/libraries/openshift/openshift.py'
```

Comment:
@ashetty1 this is my setup path where i ran all the test cases. This path is mutable and i am going to update in README file as commented in the next comment on README file